workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/UnitySchemaFile.cs

```diff
@@ -8,6 +9,7 @@ namespace Improbable.Gdk.CodeGenerator
     /// <summary>
     ///     Represents a schema file.
     /// </summary>
+    [DebuggerDisplay("{CanonicalName}")]
```

Comment:
Intended?

Reply:
Ah, this made debugging easier. May as well leave it.